Packs/FeedMISP/.secrets-ignore

```diff
@@ -1 +1,3 @@
-https://x.x.x.x
\ No newline at end of file
+https://x.x.x.x
+2001:0db8:85a3:0000:0000:8a2e:0370:7334
+2001:0db8:85a3:0000:0000:8a2e:0370:7334/64
```

Comment:
The secret is from
https://en.wikipedia.org/wiki/IPv6_address#:~:text=An%20example%20of%20an%20IPv6,the%20representation%20of%20IPv6%20addresses.